packages/account/src/Sections/Profile/PhoneVerification/__test__/confirm-phone-number.spec.tsx

```diff
@@ -141,7 +141,7 @@ describe('ConfirmPhoneNumber', () => {
     });
 
     it('should make both buttons disabled if next_otp_request text is provided', async () => {
-        (usePhoneNumberVerificationSetTimer as jest.Mock).mockReturnValue({ next_otp_request: '60 seconds' });
+        (usePhoneNumberVerificationSetTimer as jest.Mock).mockReturnValue({ next_request_time: 60 });
```

Comment:
can we specify which timers are this?
for next_otp_request, is it for email otp or phone otp?
next_request_time is it for email request timer or or phone request timer?,